TCMSApp/src/client/app/dashboard/dashboard.route.js

```diff
@@ -16,7 +16,7 @@
             {
                 state: 'dashboard',
                 config: {
-                    url: '/',
+                    url: '/dashboard',
                     templateUrl: 'app/dashboard/dashboard.html',
                     controller: 'DashboardController',
                     controllerAs: 'vm',
```

Comment:
maybe something like dashboardVm would be better. 
Why? So that we can access parent controller by its name from any child controller